docs/arithmetics.rst

```diff
@@ -26,7 +26,7 @@ Arithmetic operators
     print(2 % 3)   # 2
     print(-2 % 3)  # 1
 
-  For C-style mod, please use ``ti.raw_mod``:
+  For C style mod, please use ``ti.raw_mod``:
```

Comment:
```suggestion
  For C-style mod, please use ``ti.raw_mod``:
```

Reply:
FYI L21: `The ``%`` operator in Taichi follows the Python style instead of C style`, should we change that too?

Reply:
That one is fine.